### PR TITLE
Adds information or requeueing healing processes

### LINKED ISSUE #529: [Release] Update docs for RELEASE.2022-08-11T04-37-28Z
Release [RELEASE.2022-08-11T04-37-28Z](https://github.com/minio/minio/releases/tag/RELEASE.2022-08-11T04-37-28Z) has changes that impact the docs:

- [x] The healing process for buckets or sites with replication is now prompted when running `mc stat`, `mc cat`, or `mc ls`. Update the site replication and bucket replication docs. [PR #15501](https://github.com/minio/minio/pull/15501)
- [x] ~New feature allows you to enable object locking when creating a bucket [PR #15505](https://github.com/minio/minio/pull/15505)~ - This is for helm, will not document in the current docs.

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #529

Release RELEASE.2022-08-11T04-37-28Z optimized when site healing requeues. 
Updated bucket replication and site replication docs to reflect this change.

### CODE REVIEW

File: source/administration/bucket-replication.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ replication while adding the following MinIO-only features:
   buckets.
 
 - Multi-Site replication of objects between three or more MinIO deployments
-  (New in :minio-release:`RELEASE.2021-09-23T04-46-24Z`).
+  (Available since :minio-release:`RELEASE.2021-09-23T04-46-24Z`).
```
Comment (ravindk89) [inline source/administration/bucket-replication.rst:78]: We can just drop this, it's been more than a year.

File: source/administration/bucket-replication.rst [hunk 2]
```diff
@@ -272,6 +272,11 @@ workers operating on that queue. MinIO continuously works to replicate and
 remove objects from the queue while scanning for new unreplicated objects to
 add to the queue. 
 
+.. versionchanged:: RELEASE.2022-08-11T04-37-28Z
+
+   Failed or pending replications requeue automatically when performing a list or any ``GET`` or ``HEAD`` API method.
```
Comment (ravindk89) [inline source/administration/bucket-replication.rst:277]: Q: does this just mean they get tossed into the queue, or do we prioritize them to the front of the queue? cc @poornas

**Review by ravindk89 — COMMENTED** [review]
A few clarifying questions. Will wait on LGTM pending the other refinements we discussed offline.

File: source/administration/bucket-replication.rst (hunk 1 above)
Comment (djwfyi) [reply source/administration/bucket-replication.rst:78]: dropped

**Review by ravindk89 — APPROVED** [review]
LGTM, though I'd like clarity on the queue behavior standing.

File: source/administration/bucket-replication.rst (hunk 2 above)
Comment (djwfyi) [reply source/administration/bucket-replication.rst:276]: Thanks @poornas.
Will merge as is, then.

**Pull request merged**